docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -138,6 +139,37 @@ Type | Key | value | Description
+| &#xfeff;                         | gamma_initializer           | (weight_initializer) | gamma initializer                                                     |
+| &#xfeff;                         | momentum                    | (float)              | momentum                                                              |
+| `ML_TRAIN_LAYER_TYPE_CONV2D`     |                             |                      | 2D convolution layer                                                  |
+| &#xfeff;                         | filters                     | (integer)            | number of filters                                                     |
+| &#xfeff;                         | kernel_size                 | (integer array)      | comma-separated integers for kernel size,  `height, width` respectively |
+| &#xfeff;                         | stride                      | (integer array)      | comma-separated integers for strides, `height, width`, respectively   |
+| &#xfeff;                         | padding                     | (integer array)      | comma-separated integers for padding, `height, width`, respectively   |
+| `ML_TRAIN_LAYER_TYPE_POOLING2D`  |                             |                      | 2D pooling layer                                                      |
+| &#xfeff;                         | pooling                     | (categorical)        | type of pooling                                                       |
+| &#xfeff;                         |                             | max                  | max pooling                                                           |
+| &#xfeff;                         |                             | average              | average pooling                                                       |
+| &#xfeff;                         |                             | global_max           | global max pooling                                                    |
+| &#xfeff;                         |                             | global_average       | global average pooling                                                |
+| `ML_TRAIN_LAYER_TYPE_FLATTEN`    |                             |                      | Flatten layer                                                         |
+| `ML_TRAIN_LAYER_TYPE_ACTIVATION` |                             |                      | Standalone, simple activation layer                                   |
+| `ML_TRAIN_LAYER_TYPE_ADDITION`   |                             |                      | Addition layer                                                        |
+| `ML_TRAIN_LAYER_TYPE_CONCAT`     |                             |                      | Concatenation layer                                                   |
+| `ML_TRAIN_LAYER_TYPE_MULTIOUT`   |                             |                      | Muliout layer, repeats output to the number of layers                 |
```

Comment:
There is a spelling mistake, you have again missed editing it. 

```suggestion
| `ML_TRAIN_LAYER_TYPE_MULTIOUT` | | | Multiout layer, repeats output to the number of layers |

Reply:
Thank you, it is fixed :)